.gitignore

```diff
@@ -17,3 +17,4 @@ datasets/
 /topicalguide/settings.py
 /working/
 tg_models.png
+/tools/stemmer/
```

Comment:
Yes, I don't think so either. I have to put that in .gitignore or it causes
problems for me. Is there a way to ignore that file from the pull request?
Because I don't think I can ignore the .gitignore within the .gitignore. Or
can I?

Anyways, yeah, the .gitignore is not supposed to be in the pull request,
but I don't know how to get rid of it.

On Wed, Feb 10, 2016 at 8:52 AM, Connor Cook <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87#discussion_r52476365>
> :
>
> > @@ -17,3 +17,4 @@ datasets/
> >  /topicalguide/settings.py
> >  /working/
> >  tg_models.png
> > +/tools/stemmer/
>
> I don't think we want to have git totally ignore the stemmer directory. We
> do want it to ignore the Snowball and stemmer binaries inside that
> directory, though (users should build this, as directed in the README).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87/files#r52476365>.
>